src/ComputeSharp.Graphics/Buffers/ConstantBuffer{T}.cs

```diff
@@ -24,7 +24,7 @@ public sealed class ConstantBuffer<T> : HlslBuffer<T> where T : unmanaged
         /// </summary>
         /// <param name="device">The <see cref="GraphicsDevice"/> associated with the current instance</param>
         /// <param name="size">The number of items to store in the current buffer</param>
-        internal ConstantBuffer(GraphicsDevice device, int size) : base(device, size, size * GetPaddedSize() * 16, BufferType.Constant) { }
+        internal ConstantBuffer(GraphicsDevice device, int size) : base(device, size, size * GetPaddedSize(), BufferType.Constant) { }
```

Comment:
Just so I understand, why have you removed the `* 16` here?

Reply:
I switch GetPaddedSize to return the real size, not the size / 16. It isn't reference by anything outside this type so it isn't a breaking change, and allows indexing to work more intuitively

Reply:
Ah yeah that makes total sense, I see now. Thanks! 👍